src/WebView.php

```diff
@@ -544,7 +544,7 @@ protected function renderBodyEndHtml(bool $ajaxMode): string
      */
     public function getTitle(): string
     {
-        return $this->title;
+        return isset($this->title) ? $this->title : '';
```

Comment:
```suggestion
        return $this->title ?? null;
```

Reply:
That is not an option with uninitialized typed properties.

Reply:
https://3v4l.org/8dfRM

Reply:
But yes, return type should be `?string` in that case.